### PR TITLE
Bumping golang version for origin

### PR DESCRIPTION
**Pull request by jupierce**
Bumping origin golang requirement to 1.8 for kube 1.7

### CODE REVIEW

**Comment by deads2k** [comment]
@stevekuznetsov there was a check I disabled during the rebase https://github.com/openshift/origin/issues/15270 .  Is this the right time to fix it?

/approve

**Comment by stevekuznetsov** [comment]
@deads2k sure, I added the right version to our AMI but did not have time to fix the shell check. Added here.

File: origin.spec [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # this is the version we obsolete up to. The packaging changed for Origin
 # 1.0.6 and OSE 3.1 such that 'openshift' package names were no longer used.
 %global package_refector_version 3.0.2.900
-%global golang_version 1.6.2
+%global golang_version 1.8.3
```
Comment (smarterclayton) [inline origin.spec]: Do we *actually* require 1.8.3, or any 1.8?

Comment (jupierce) [inline origin.spec]: 1.8.3 is tagged for RHEL, but I don't think we technically need that exact version. Will change.

**Comment by stevekuznetsov** [comment]
@jupierce you nuked my commit ...

**Comment by jupierce** [comment]
@stevekuznetsov sorry, didn't realize you were linked.

**Comment by stevekuznetsov** [comment]
FYI my changes were pulled in from a different PR -- this one is now self-sufficient

**Pull request closed without merging**